apps/wallet/src/test-utils/vault.ts

```diff
@@ -17,13 +17,14 @@ export const testEd25519Serialized = Object.freeze({
         'a3R0jvXpEziZLHsbX1DogdyGm8AK87HScEK+JJHwaV99nEpOfYblbYS3ci9wP2DT5YZtE3e4v/HBsN39kRz60A==',
 } as const);
 export const testEd25519 = fromExportedKeypair(testEd25519Serialized);
-export const testEd25519Address = testEd25519.getPublicKey().toSuiAddress();
+export const testEd25519Address = '0x3f75982331cc2fb8deb47068a711d73a40542757';
```

Comment:
Why the hard-code?

Reply:
No particular reason, had to add the `0x` and just hardcoded it. Will change it